Extension/package.json

```diff
@@ -5638,6 +5635,11 @@
                             "processId"
                         ],
                         "properties": {
+                            "program": {
+                                "type": "string",
+                                "description": "%c_cpp.debuggers.program.description%",
+                                "default": "${workspaceRoot}/program.exe"
```

Comment:
Can you change the `workspaceRoot` references to `workspaceFolder`? `workspaceRoot` is the old variable name.

Reply:
I fixed them all in the file. There were a lot of `workspaceRoot` values.